virt/virtual_machines/vm_networking/virt-attaching-vm-to-external-network.adoc

```diff
@@ -0,0 +1,80 @@
+:_content-type: ASSEMBLY
+[id="virt-attaching-vm-to-external-network"]
+= Attaching a virtual machine to an external network
+include::_attributes/common-attributes.adoc[]
+:context: virt-attaching-vm-to-external-network
+
+toc::[]
+
+{VirtProductName} is installed with a default internal `pod` network. If you want to attach VMs to an external network, however, you must create a Linux _network bridge_.
+
+Depending on the type of networks you are using, it may be necessary to update your xref:../../../virt/node_network/virt-updating-node-network-config.adoc#virt-about-nmstate_virt-updating-node-network-config[node networking configuration].
```

Comment:
This is redundant. Jean-Francois said that configuring the Linux bridge is the only configuration that is required. So we can remove this sentence.

Reply:
Yes, again this is an older version of the file. The current version reads:

{VirtProductName} is installed with a default internal `pod` network. You update your xref:../../../virt/node_network/virt-updating-node-network-config.adoc#virt-about-nmstate_virt-updating-node-network-config[node networking configuration] depending on the type of network to which you want to connect.

My understanding is that the Linux bridge is mandatory but there may be other configuration steps required for connection to additional networks beyond having a Linux bridge. Maybe that is not correct, but I would check with JF first. if it's fine with him, fine with me....

Reply:
Removed/resolved.